src/aat/resources/features/F-107/S-927_Create_Case_Data.td.json

```diff
@@ -0,0 +1,104 @@
+{
+	"_guid_": "S-927_Create_Case_Data",
+  "_extends_": "F-107_Test_Data_Base",
+  "request": {
+    "body": {
+      "event_token": "${[scenarioContext][siblingContexts][F-107_Case_Data_Create_Token_Creation][testData][actualResponse][body][token]}"
+    }
+  },
+
+  "expectedResponse": {
+    "_extends_": "Common_201_Response",
+    "headers": {
+      "Keep-Alive": "[[ANYTHING_PRESENT]]",
+      "Connection": "[[ANYTHING_PRESENT]]",
+      "Transfer-Encoding" : "[[ANYTHING_PRESENT]]",
+      "vary" : "[[ANYTHING_PRESENT]]",
+      "Content-Encoding": "[[ANYTHING_PRESENT]]"
+    },
+    "body": {
+      "id": "[[ANYTHING_PRESENT]]",
+      "jurisdiction": "BEFTA_MASTER",
+      "case_type_id": "FT_MasterCaseType",
+      "created_date": "[[ANYTHING_PRESENT]]",
+      "last_modified": "[[ANYTHING_PRESENT]]",
+      "last_state_modified_date": "[[ANYTHING_PRESENT]]",
+      "state": "CaseCreated",
+      "version": "[[ANYTHING_PRESENT]]",
+      "security_classification": "PUBLIC",
+      "security_classifications": "[[ANYTHING_PRESENT]]",
+      "case_data": {
+        "AddressField": {
+          "AddressLine1": "[[ANYTHING_PRESENT]]",
+          "AddressLine2": "[[ANYTHING_PRESENT]]",
+          "AddressLine3": "[[ANYTHING_PRESENT]]",
+          "Country": "[[ANYTHING_PRESENT]]"
+        },
+        "TextField": "[[ANYTHING_PRESENT]]",
+        "OrganisationPolicyField2": {
+          "OrgPolicyCaseAssignedRole": "[Defendant]",
+          "OrgPolicyReference": "ref",
+          "Organisation": {
+            "OrganisationID": "[[ANYTHING_PRESENT]]",
+            "OrganisationName": "[[ANYTHING_PRESENT]]"
+          }
+        },
+        "TextAreaField": "[[ANYTHING_PRESENT]]",
+        "OrganisationPolicyField1": {
+          "OrgPolicyCaseAssignedRole": "[Claimant]",
+          "OrgPolicyReference": "ref",
+          "Organisation": {
+            "OrganisationID": "[[ANYTHING_PRESENT]]",
+            "OrganisationName": "[[ANYTHING_PRESENT]]"
+          }
+        }
+      },
```

Comment:
The specific values for many of [[ANYTHING_PRESENT]] placeholders should better be checked in exact terms. This is particularly required at least for values in OrganisationPolicy fields and their sub-fields.

Reply:
Added the actual expected values in the expecteResponse, specifically for OrganisationPolicy.